src/components/home/Events.jsx

```diff
@@ -1,20 +1,44 @@
 // Material UI Imports
-import Box from '@mui/material/Box';
 import Container from '@mui/material/Container';
 import Grid from '@mui/material/Grid';
+import Link from '@mui/material/Link';
 import Paper from '@mui/material/Paper';
 import Typography from '@mui/material/Typography';
 
 const leftSidePaperStyles = {
   elevation: 3,
   sx: {
-    mr: '20%',
+    mr: '30%',
     mb: '40px',
-    borderRadius: '30px',
-    background: '#d9d9d9'
+    borderRadius: '20px',
+    background: 'rgba(217, 217, 217, 0.4)'
   }
 };
 
+const DiscordLink = () => {
+  return (
+    <Link href="https://discord.gg/h2pTvDbJeu" target="_blank" rel="noopener noreferrer">
```

Comment:
The links on the Volunteer page don't have underlines, so we can remove them here to keep it consistent.